events/api/src/test/java/org/projectnessie/events/api/TestCommitMeta.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.events.api;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Most of Nessie code uses `AssertJ` ;)

Reply:
Yeah, I was reluctant to add the dependency just for 3 cases so far: assert equals, assert true and assert false. But OK to add it.

Reply:
I'd say let's add it - it's not a heavy artifacts and the build as a whole pulls it anyway - and we get uniformity of test style across the codebase (TBH, I do not often remember how to write assertions in the plain JUnit way anymore :) )